docsteady/__init__.py

```diff
@@ -368,7 +368,14 @@ def generate_vcd(format, jiradb, vcduser, vcdpwd, username, password, sql, spec,
             if 'req_doc_id' in ve_model[ve].keys():
                 req_dict[ve_model[ve]['req_id']]['reqDoc'] = ve_model[ve]['req_doc_id']
             ve_dict[ve_long_name[0]] = tmp_ve
-        vcd_dict = [ve_dict, req_dict, [], Config.CACHED_TESTCASES]
+        # Not sure why the ve_dict is keyed on Requirement with a version -
+        # everying wants verificaiton element so remaking it (wom)
```

Comment:
typo: "everying"

You don't need to sign a comment -- git blame will tell us who made it.